### ISSUE TITLE
Upgrading node-sass to version 4?

### ISSUE THREAD

**Issue by neoromantic** [nathantreid/meteor-css-modules#93]
Is there any particular reason that meteor-css-modules requires node-sass 3 instead of current version (which is 4)?

There are several unrelated reasons of which I have to use node-sass4. So i get this warning:

```
WARNING: version mismatch for node-sass; installed version is 4.5.1, but version 3.x is required by nathantreid:css-modules)
```

Things works good with 4.5.1. Is there a reason to require v3?

**Comment by akanix42** [comment]
No, it seems to work fine - I just need to release a patch that adds v4 to the supported versions.

**Comment by neoromantic** [comment]
That’s great! Waiting for that patch :)




Sergey.

**Comment by akanix42** [comment]
This has been fixed in v2.5.2.
